Libraries/LibJS/Interpreter.cpp

```diff
@@ -55,6 +56,7 @@ Interpreter::Interpreter()
     m_error_prototype = heap().allocate<ErrorPrototype>();
     m_date_prototype = heap().allocate<DatePrototype>();
     m_number_prototype = heap().allocate<NumberPrototype>();
+    m_boolean_prototype = heap().allocate<BooleanPrototype>();
```

Comment:
You need to track this in: Interpreter::gather_roots

Reply:
Thanks!